Exercise_4/docs/part-3.md

```diff
@@ -0,0 +1,176 @@
+
+??? question "Knowledge Check: Which element(s) of this profile must be supported in order to be US Core Compliant?"
+    `Medication.code` element
+
+Going back to our **smoking example** take a look here: HL7.FHIR.US.CORE\US Core Smoking Status Observation Profile - FHIR® v4.0.1 [https://hl7.org/fhir/us/core/StructureDefinition-us-core-smokingstatus.html#:~:text=Resource%20Profile%3A%20US%20Core%20Smoking%20Status%20Observation%20Profile,as%20of%202019-05-21T00%3A00%3A00%2B00%3A00%20%204%20more%20rows%20](https://hl7.org/fhir/us/core/StructureDefinition-us-core-smokingstatus.html#:~:text=Resource%20Profile%3A%20US%20Core%20Smoking%20Status%20Observation%20Profile,as%20of%202019-05-21T00%3A00%3A00%2B00%3A00%20%204%20more%20rows%20)
+
+![Smoking core profile](img/smoking_core.png)
+
+We can see here a list of required elements this profile must support.
+
+??? question "Knowledge Check: How many elements are required and what are they?"
+    7 required elements. They include a `status`, a category code of `social-history`, a code for smoking observation, a patient, when the observation occurred, a result value code for smoking status
+
+??? question "Knowledge Check: Where would I find the reference ID of the patient?"
+    `Subject code`
+
+Let’s take a look at the raw unformatted **JSON code** for the resource:
+
+??? note "Sample JSON code of the Smoking Resource"
+    {
+    "resourceType": "Observation",
+    "id": "056f4fe8-cd0a-db01-9d8b-0b3bf098368c",
+    "meta": {
+        "profile": [ "http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus" ]
+    },
+    "status": "final",
+    "category": [{
+        "coding": [{
+            "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+            "code": "social-history",
+            "display": "social-history"
+        }]
+    }],
+    "code": {
+        "coding": [{
+        "system": "http://loinc.org",
+        "code": "72166-2",
+        "display": "Tobacco smoking status NHIS"
+        }],
+        "text": "Tobacco smoking status NHIS"
+    },
+    "subject": { "reference": "urn:uuid:c37a6b5c-a13c-835d-bfe2-e215df91cb3c" },
+    "encounter": { "reference": "urn:uuid:0f7f8554-0896-65dd-7bed-fe85762791dd" },
+    "effectiveDateTime": "2012-07-25T10:07:28-04:00",
+    "issued": "2012-07-25T10:07:28.029-04:00",
+    "valueCodeableConcept": {
+        "coding": [{
+        "system": "http://snomed.info/sct",
+        "code": "266919005",
+        "display": "Never smoker"
+        }],
+        "text": "Never smoker"
+    }
+    }
+Check out this patient profile here: HL7.FHIR.US.CORE\Some Day Smoker Example - JSON Representation - FHIR® v4.0.1
+
+??? question "Knowledge Check: What is the name of the patient"
+    Amy Shaw
+??? question "Knowledge Check: Is she a smoker and if so, what kind?"
+    Current smoker
```

Comment:
These are confusing because the questions are about the "Some Day Smoker Example" linked above, but the sample JSON immediately above that link has different values!